src/MICoreUnitTests/NdkVersionTests.cs

```diff
@@ -52,14 +52,14 @@ public void TestNdkToolVersion2()
         }
 
         [Fact]
-        public void TestNdkReleaseId1()
+        public void TestNdkReleaseIdReleaseFile()
         {
             NdkReleaseId r;
             Assert.True(NdkReleaseId.TryParse("r1", out r));
             Assert.Equal(r.ToString(), "r1");
 
             Assert.True(NdkReleaseId.TryParse("r1a", out r));
-            Assert.Equal(r.ToString(), "r1a");
+            Assert.Equal(r.ToString(), "r1");
```

Comment:
This is the corresponding change in the tests to get rid of 'a'.

BTW all these tests have the arguments passed to Assert.Equal the wrong way (it's (expected, actual)). I didn't want to introduce more changes. I can change them if you all agree.